gapic/templates/%namespace/%name_%version/%sub/services/%service/async_client.py.j2

```diff
@@ -24,6 +25,7 @@ from google.api_core import retry_async as retries
 from google.auth import credentials as ga_credentials   # type: ignore
 from google.oauth2 import service_account              # type: ignore
 
+{{ shared_macros.add_google_api_core_version_header_import(service) }}
```

Comment:
Why not just pass down `service.version` here? Same is true for `add_api_version_header_to_metadata` if that's the only attribute we need from service for these two macros.

Reply:
Fixed in https://github.com/googleapis/gapic-generator-python/pull/1999/commits/f150e6783b0cc3c72b44a79cf6865075c1fcf96e